cob_bringup/drivers/mimic.launch

```diff
@@ -1,11 +1,5 @@
 <?xml version="1.0"?>
 <launch>
-
-	<arg name="rotation" default="0"/>
-
-	<!-- start mimic -->
-	<node pkg="cob_mimic" type="mimic_node.py" name="mimic" cwd="node" respawn="false">
-		<param name="rotation" value="$(arg rotation)"/>
```

Comment:
@ipa-bnm @ipa-fmw @ipa-nhg
just to confirm....there is no need for `rotation` argument on the robots anymore?
I remember @ipa-nhg mentioning that some head displays are configured different than other...on which one did you test?

simulation is fine...